js/Snake.js

```diff
@@ -56,7 +56,16 @@ class Snake extends GridChild { // eslint-disable-line no-unused-vars
 		if (cell.child instanceof Food) {
 			cell.child.destroy();
 			this.size++;
+
+			var body = document.querySelector('body');
+
+			body.classList.add('flash');
```

Comment:
Let's maybe move this onto grid and make it generic -- that way we can flash from other places too if we want to.

Something like this:

```js
// Snake.js
// ...in this method
this.grid.flash('food');

// Grid.js
// ...as a new method
flash(type) {
  const body = document.querySector('body');
  const flashClass = `flash-${type}`;
  body.classList.add(flashClass);
  setTimeout(function () {
    body.classList.remove(flashClass);
  }, options.flashDuration); // add this to the global options object in main.js instead of hardcoding as 250
}
```

It may be worth moving some of the flash logic even out to `HTMLRenderer.js` as the flash code right as written has to do with the rendering and not the grid state -- that'd be more in line with the [Single Responsibility Principle](https://en.wikipedia.org/wiki/Single_responsibility_principle). It'd look something like...

```js
// Snake.js
this.grid.flash('food');

// Grid.js
flash(type) {
  this.renderer.flash(type);
}

// HTMLRenderer.js
flash(type) {
  // ... logic goes here
}
```

I don't love the idea of making a chain of calls like that but sometimes it's best to go ahead and do those things and wait for the more elegant or appropriate solution to become clear as the program grows.